native/package.json

```diff
@@ -11,20 +11,21 @@
     "android:build": "react-native run-android",
     "android": "npm run android:emulator & npm run android:build",
     "test": "jest --coverage",
-    "storybook": "storybook start -p 7007 --root ../"
+    "storybook": "storybook start -p 7007 --skip-packager | react-native start --watchFolders $PWD/.."
```

Comment:
Storybook used to run the RN packager for us but their usage doesn't work with the new RN (see https://github.com/storybooks/storybook/issues/4179).

So, now we run the packager ourselves. `$PWD/..` is because Watchman won't accept a relative path (i.e. `..`), so this was my ~hack~ workaround.